### PR TITLE
Allow "cylc graph --help" in absence of X environment.

### PR DESCRIPTION
**Pull request by hjoliver**
(what it says).

### CODE REVIEW

**Comment by matthewrmshin** [comment]
@arjclark please sanity check.

File: bin/cylc-graph [hunk 1]
```diff
@@ -204,6 +207,9 @@ def main():
 
     (options, args) = parser.parse_args()
 
+    if no_x:
+        sys.exit(1)
```
Comment (arjclark) [inline bin/cylc-graph]: Could we just `return`/`sys.exit(0)` if `--help` is used? Doesn't seem like it should be a non-zero exit code as its successfully printing help details...

Comment (hjoliver) [inline bin/cylc-graph]: `--help` causes the script to exit before this point... and I just confirmed this really is the case by faking a gtk ImportError.

Comment (arjclark) [inline bin/cylc-graph]: Cool, good to go then

**Pull request merged**